hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMBlockProtocolServer.java

```diff
@@ -318,12 +322,17 @@ public List<DeleteBlockGroupResult> deleteKeyBlocks(
       results.add(new DeleteBlockGroupResult(bg.getGroupID(), blockResult));
     }
     auditMap.put("KeyBlockToDelete", keyBlocksInfoList.toString());
+    auditMap.put("NumOfKeyBlocksToDelete", String.valueOf(keyBlocksInfoList.size()));
     if (e == null) {
       AUDIT.logWriteSuccess(
           buildAuditMessageForSuccess(SCMAction.DELETE_KEY_BLOCK, auditMap));
+      BG_DELETION_AUDIT.logWriteSuccess(
```

Comment:
We do not need another audit log if above are there. this is to log external request. May be can check if comming or not.